src/pages/registration/resource_type_tab/components/JournalField.tsx

```diff
@@ -147,6 +152,14 @@ export const JournalField = ({ confirmedContextType, unconfirmedContextType }: J
             renderOption={(props, option, state) => (
               <PublicationChannelOption key={option.id} props={props} option={option} state={state} />
             )}
+            ListboxComponent={AutocompleteListboxWithExpansion}
+            ListboxProps={
+              {
+                hasMoreHits: !!journalOptionsQuery.data?.totalHits && journalOptionsQuery.data.totalHits > searchSize,
+                onShowMoreHits: () => setSearchSize(searchSize + defaultChannelSearchSize),
+                isLoadingMoreHits: journalOptionsQuery.isFetching && !journalOptionsQuery.isPending,
```

Comment:
Denne var litt vanskelig å lese, mulig fordi jeg ikke er godt nok kjent med Tanstack. Men etter å ha gravd litt så forstår jeg det som at du bruker `isFetching` for å sjekke at "noe hentes", og `!isPending` for å sjekke at "noe har blitt hentet før"? Ser i dokumentasjonen at første gangs lasting har en derived boolean `isLoading = isFetching && isPending` (`isLoading` het `isInitialLoading` [tidligere](https://github.com/TanStack/query/discussions/4252)). Kunne vi lagd vår egen extension metode på useQuery  `isNonInitialLoading = isFetching && !isPending` for å gjøre det litt mer leselig hva som foregår?
Er jo ikke galt slik det er nå, så du får vurdere selv om vi skal gjøre noe med det.

Reply:
Tanken her var at "Vis flere"-knappen bare skal ha loading state (spinner) når brukeren venter på flere treff av søket man allerede har gjort. Altså at den ikke spinner når bruker gjør ett nytt søk.

Men ser nå at det ikke virker helt som tenkt, så må se litt mer på denne, og sikkert finne en litt annen løsning 🤔